internal/clientinfo/mdns_services.go

```diff
@@ -67,4 +67,16 @@ var services = [...]string{
 
 	// Merlin
 	"_alexa._tcp",
+
+	// Newer Android TV devices
+	"_androidtvremote2._tcp.local.",
```

Comment:
I don't think we need `.local.` suffix, please remove them.

Reply:
All of the other services except `_alexa._tcp` include the `.local.` suffix. Do you want me to remove `.local.` for **all** of the services defined here?